### ISSUE TITLE
Can't participate in slack channel

### ISSUE THREAD

**Issue by nobu0000** [facebook/fbctf#585]
When I tried to participate in slack channel via email, it failed. Does fbctf team still accept entering slack channel?

**Comment by stevcoll** [comment]
@nobu0000 Yes it's still open, I've seen a few people join lately. What's the exact error you're getting?

**Comment by nobu0000** [comment]
When I submit email address, it said... "Failed! invite_limit_reached"

**Comment by stevcoll** [comment]
I found [this](https://github.com/HackBrexit/hackbrexit.github.io/issues/24#issuecomment-245832917). We will look into it further and provide an update, thanks!

**Comment by stevcoll** [comment]
@nobu0000 Is this an invite you're trying to accept, or are you inviting other people?

**Comment by stevcoll** [comment]
@nobu0000 Apparently it's a Slack limit on invitations. We have reached out to support to try to get an increase.

**Comment by sbadakhc** [comment]
I can confirm that this is still the case as of 23/01/2018.

**Comment by zados** [comment]
Any updates on this?
